flume-ng-sinks/flume-ng-elasticsearch-sink/src/main/java/org/apache/flume/sink/elasticsearch/ElasticSearchSink.java

```diff
@@ -419,7 +413,7 @@ private long parseTTL(String ttl) {
       } else if (matcher.group(2).equals("d")) {
         return TimeUnit.DAYS.toMillis(Integer.parseInt(matcher.group(1)));
       } else if (matcher.group(2).equals("w")) {
-        return TimeUnit.DAYS.toMillis(7 * Integer.parseInt(matcher.group(1)));
```

Comment:
Note to reviewers: SpotBugs was reporting `ICAST_INTEGER_MULTIPLY_CAST_TO_LONG`.